src/index.html

```diff
@@ -4,13 +4,51 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class= "search-form"
+      method="get"
+      data-qa="big"
+      >
+      <img
+          class="big-icon"
+          src="images/Search.svg"
+          alt="big-icon"
+      >
```

Comment:
Need to fix indentation size.
```suggestion
      <img
        class="big-icon"
        src="images/Search.svg"
        alt="big-icon"
      >
```